lib/osm_community_index.rb

```diff
@@ -0,0 +1,26 @@
+module OsmCommunityIndex
+  def self.add_to_i18n
+    communities = Community.all
+    files = Dir.glob(Rails.root.join("node_modules/osm-community-index/i18n/*"))
+    files.each do |file|
+      locale = File.basename(file, ".yaml")
+      community_locale_yaml = YAML.safe_load(File.read(file))[locale]
+      # rails wants en-GB but osm-community-index has en_GB
+      locale_rails = locale.tr("_", "-")
```

Comment:
Presumably this affects more than just `en-GB` so maybe the comment should be more generic, like `language-COUNTRY` or something?

Are we sure this is the only mismatch? Has anybody done a complete cross check between our language list and the OCI one?

Reply:
Updated the comment, and move the complete cross check discussion to #3653